specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/examples/Endpoint/getModels.json

```diff
@@ -0,0 +1,135 @@
+            "capabilities": {
+              "fineTune": "true",
+              "inference": "false",
+              "scaleType": "Manual",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "finetuneCapabilities": {
+              "scaleType": "Manual,Standard",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "deprecation": {
+              "fineTune": "2024-07-05T00:00:00+00:00",
+              "inference": "2024-07-05T00:00:00+00:00"
+            },
+            "systemData": {
+              "createdBy": "Microsoft",
+              "createdByType": null,
+              "lastModifiedBy": "Microsoft",
+              "lastModifiedByType": null
+            },
+            "format": "OpenAI",
+            "name": "ada",
+            "version": "1",
+            "source": null,
+            "callRateLimit": null
+          },
+          {
+            "baseModel": null,
+            "isDefaultVersion": false,
+            "skus": [
+              {
+                "name": "Standard",
+                "usageName": "OpenAI.Standard.babbage",
+                "deprecationDate": "2024-03-01T00:00:00+00:00",
+                "capacity": {
+                  "minimum": null,
+                  "maximum": 10000,
+                  "step": null,
+                  "default": 120
+                },
+                "rateLimits": [
+                  {
+                    "count": 1.0,
+                    "renewalPeriod": 10.0,
+                    "rules": []
+                  },
+                  {
+                    "count": 1000.0,
+                    "renewalPeriod": 60.0,
+                    "rules": []
+                  }
+                ]
+              }
+            ],
+            "maxCapacity": 3,
+            "capabilities": {
+              "fineTune": "true",
+              "inference": "false",
+              "scaleType": "Manual",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "finetuneCapabilities": {
+              "scaleType": "Manual,Standard",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "deprecation": {
+              "fineTune": "2024-07-05T00:00:00+00:00",
+              "inference": "2024-07-05T00:00:00+00:00"
+            },
+            "systemData": {
+              "createdBy": "Microsoft",
+              "createdByType": null,
+              "lastModifiedBy": "Microsoft",
+              "lastModifiedByType": null
+            },
+            "format": "OpenAI",
+            "name": "babbage",
+            "version": "1",
+            "source": null,
+            "callRateLimit": null
```

Comment:
sending or returning explicit nulls is usually discouraged, I'd recommend not using x-nullable, but just making the property optional, or returning a default value

Reply:
OK the example is auto genereated, I will update those to a more meaningful example.